libraries/GCS_MAVLink/GCS.h

```diff
@@ -129,6 +129,13 @@ class GCS_MAVLINK
     // call to reset the timeout window for entering the cli
     void reset_cli_timeout();
 
+    bool is_usb() { return chan == MAVLINK_COMM_0; }
```

Comment:
What about `is_console`? I don't like it very much either, but I think that is the only safe assumption we can make. Even high bandwidth isn't correct, because in a Linux all channels may be high bandwidth.

We shouldn't change now, but is there any reason not to base the check on the UART baudrate instead of anything else?